src/adagrams.js

```diff
@@ -1,8 +1,103 @@
 const Adagrams = {
+  letterInfo: {
+    "A": {count: 9, points: 1},
+    "B": {count: 2, points: 3},
+    "C": {count: 2, points: 3},
+    "D": {count: 4, points: 2},
+    "E": {count: 12, points: 1},
+    "F": {count: 2, points: 4},
+    "G": {count: 3, points: 2},
+    "H": {count: 2, points: 4},
+    "I": {count: 9, points: 1},
+    "J": {count: 1, points: 8},
+    "K": {count: 1, points: 5},
+    "L": {count: 4, points: 1},
+    "M": {count: 2, points: 3},
+    "N": {count: 6, points: 1},
+    "O": {count: 8, points: 1},
+    "P": {count: 2, points: 3},
+    "Q": {count: 1, points: 10},
+    "R": {count: 6, points: 1},
+    "S": {count: 4, points: 1},
+    "T": {count: 6, points: 1},
+    "U": {count: 4, points: 1},
+    "V": {count: 2, points: 4},
+    "W": {count: 2, points: 4},
+    "X": {count: 1, points: 8},
+    "Y": {count: 2, points: 4},
+    "Z": {count: 1, points: 10},
+  },
+
+  drawCount: 10,
+  
   drawLetters() {
-    // Implement this method for wave 1
+    const letterInfo = this.letterInfo;
+    const letterPool = [];
+    for (let letter in letterInfo) {
+      const letterCount = letterInfo[letter].count
+      for (let i = 0; i < letterCount; i++){
+        letterPool.push(letter); 
+      }
+    }
+    // console.log(letterPool);
+
+    const randomIndices = [];
+
+    while (randomIndices.length < this.drawCount) {
+      const randomIndex = Math.floor(Math.random() * letterPool.length);
+      if (!randomIndices.includes(randomIndex)) {
+        randomIndices.push(randomIndex);
+      }
+    }
```

Comment:
Nicely done. If you ever think about refactoring this, I think it might be interesting to refactor this `while` loop so that: instead of pushing `randomIndex` into `randomIndices` until `randomIndices` has a length of ten, and then mapping over` randomIndices in order to populate `drawnLetters` ... Consider using `randomIndices` on `letterPool` to populate/push into `drawnLetters` directly